tokio/src/lib.rs

```diff
@@ -392,13 +395,16 @@ compile_error! {
     "Tokio requires the platform pointer width to be 32, 64, or 128 bits"
 }
 
-// Includes re-exports used by macros.
+// #[macro_use] is used to import the `cfg_*` macros everywhere in the crate.
+#[macro_use]
+mod util;
```

Comment:
I can't help but wonder if moving all of the macros out of the macros folder makes them harder to find?

Reply:
I would vote for this PR as it is, without the `macros` folder. 
it will spread the macros complexity towards the usage context.